manheim_cloudmapper/cloudmapper.sh

```
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# set -e
```

I'm pretty sure we want this `-e` back? As it stands right now, it appears that every command here except the final `curl` can fail, and we'll still report success to Datadog. 

We can keep this script for now, in order to get an initial version released, but since everything that's being called here is written in Python, I'd really like to see this entire script rewritten in Python, and with proper error/exception handling.

---

So my problem with the `set -e` is that `pipenv run python cloudmapper.py collect --account $ACCOUNT` can cause warnings when it cannot access certain resources in AWS, and this we throw an error and stop execution all together. 

i agree this should be moved to python eventually

---

Hmm... well that sucks. But we really don't want to silently ignore meaningful errors, especially if we're sending a success message at the end.

Can we `set -e` at the top, then `set +e` before the collect command, and `set -e` again after it, to just ignore errors from that one command? Or, even simpler, will it work right (I can't remember) if we leave the `set -e` in and then ``pipenv run python cloudmapper.py collect --account $ACCOUNT || true`` ?